tests/pytorch/mock_sparse2/test_sparse_matrix.py

```diff
@@ -10,15 +10,14 @@
 if not sys.platform.startswith("linux"):
     pytest.skip("skipping tests on win", allow_module_level=True)
 
+
 @pytest.mark.parametrize("dense_dim", [None, 4])
-@pytest.mark.parametrize("row", [[0, 0, 1, 2], (0, 1, 2, 4)])
+@pytest.mark.parametrize("row", [(0, 0, 1, 2), (0, 1, 2, 4)])
 @pytest.mark.parametrize("col", [(0, 1, 2, 2), (1, 3, 3, 4)])
 @pytest.mark.parametrize("shape", [None, (3, 5), (5, 3)])
 def test_create_from_coo(dense_dim, row, col, shape):
```

Comment:
I further suggest add test for autograd for the create ops. We only need to test gradient is correctly passed backwardly for the value array.

Reply:
I think it is not necessary to test autograd here. The creation of a sparse matrix does not have autograd since Pytorch only compute autograd on tensors. For example, 
```
A = create_from_coo(row, col, val)
B = create_from_coo(row, col, va1)
C = A + B
```
will compute autograd on `val + val1` because in our underlying code, we call the `val + val1` from libtorch. The sparse format creation itself does not have autograd.